priam/src/main/java/com/netflix/priam/aws/SDBInstanceData.java

```diff
@@ -120,12 +133,24 @@ public PriamInstance getInstance(String app, int id) {
      * @return the set of all instances in the given {@code app}
      */
     public Set<PriamInstance> getAllIds(String app) {
+        return getAllIds(app, false);
```

Comment:
Simply because using strong consistency is more expensive, and I didn't want to force clients to pay that price when they don't really need to. After thinking about it a bit more, though, I think you're right: we're operating on such a small scale that performance probably doesn't matter much relative to the benefits of getting consistent results by default. I don't have any issues changing this if you'd like me to.